oak-doc/src/site/markdown/participating.md

```diff
@@ -15,29 +15,62 @@
    limitations under the License.
   -->
 
-Participating
--------------
+# Participating
+
+## Mailing Lists
 
 The best place for Oak-related discussions is the [oak-dev@](mailto:oak-dev@jackrabbit.apache.org)
-mailing list. To subscribe, send a message to [oak-dev-subscribe@]
-(mailto:oak-dev-subscribe@jackrabbit.apache.org). 
+mailing list. To subscribe, send a message to [oak-dev-subscribe@](mailto:oak-dev-subscribe@jackrabbit.apache.org).
+
+For more details related to various mailing list have a look at http://jackrabbit.apache.org/mailing-lists.html.
+
+## Issue Tracking
 
 Use the [OAK issue tracker](https://issues.apache.org/jira/browse/OAK) to submit issues, comments 
-or patches. To subscribe to issue notifications, send a message to [oak-issues@]
-(mailto:oak-issues-subscribe@jackrabbit.apache.org). 
-
-The latest Oak sources are available on [GitHub][1]. To subscribe to commit notifications, 
-send a message to [oak-commits@] (mailto:oak-commits-subscribe@jackrabbit.apache.org).
-
-For more details related to various mailing list have a look at http://jackrabbit.apache.org/mailing-lists.html
-
-We generally follow a [CTR](https://www.apache.org/foundation/glossary.html#CommitThenReview) policy. 
-However it is up to each individual committer to pro-actively ask for a review of a patch on 
-oak-dev@ or to even call for a [RTC](https://www.apache.org/foundation/glossary.html#ReviewThenCommit). 
-Special care should be taken with backports to maintenance branches. Back ports bear a certain risk 
-of introducing regressions to otherwise stable branches. Each back ported change should be carefully 
-evaluated for its potential impact, risk and possible mitigation. It is the responsibility of each 
-committer to asses these and ask for advise or reviewing on oak-dev@ if uncertain. Whether using 
-RTC or CTR is up to the committer.
+or patches. To subscribe to issue notifications, send a message to
+[oak-issues@](mailto:oak-issues-subscribe@jackrabbit.apache.org).
+
+- On "trunk": when done with a ticket, set it to "resolved" and set "Fix Version"
+  to the next unreleased version. Our workflow does not allow re-opening a
+  closed ticket, so we close tickets only when a public release has been made
+  with that change. Until that happens, a ticket can always be re-opened
+  and further work can happen.
+- On maintenance branch (currently 1.22): re-use the existing Jira ticket an
+  just add to "Fix Version" (unless the backport is complex).
+- Add "Affects Version" and "Fix Version" as and when applicable (but not
+  otherwise).
+- Be careful not to include sensitive information (be it in descriptions,
+  attachments, or log files).
  
```

Comment:
```suggestion
 - Feel free to comment in the ticket what is being worked on. Eg to mention work starts in a certain branch B, or that the PR is ready for review, or that it is merged. It can make it easier for others to follow.

```

Should we add something along the above lines?

Reply:
PS: a whitespace sneaked into my suggestion there at the beginning of the line...